smol_podcaster.py

```diff
@@ -241,9 +245,8 @@ def main(url, name, speakers_count):
     print('Starting transcription')
     
     # function that uploads if it is a file, or just returns the url
-    url = upload_file_and_use_url(url)
-
     if not os.path.exists(raw_transcript_path):
+        url = upload_file_and_use_url(url)
```

Comment:
fyi @natolambert, it will just save time / bandwidth when re-running generations on pre-existing transcripts.